cmd/vulcan-vulners/main.go

```diff
@@ -105,8 +105,17 @@ func severity(score float32) string {
 	}
 }
 
-// vulnerabilities retrieves the vulnerabilities affecting a software component from the vulners.com API.
-func vulnerabilities(p, s, v, t string) (*report.Vulnerability, error) {
+type vulnerFinding struct {
+	Score    float32
+	Resource report.ResourcesGroup
+}
+
+// buildVulnerFinding builds a vulner Finding querying the vulners.com API. The
+// resources of the finding contain the CVE'S found for the software component.
+// The Score of the finding contains the highest score found in the all the
+// CVE's.
+func buildVulnerFinding(p, s, v, t string) (*vulnerFinding, error) {
+
```

Comment:
Can we remove the empty line? I know it's just style but just to match with the rest of the code

Reply:
Yes done